source-pendo/source_pendo/api.py

```diff
@@ -0,0 +1,123 @@
+import json
+from logging import Logger
+from typing import AsyncGenerator, Iterable
+
+from estuary_cdk.capture.common import LogCursor
+from estuary_cdk.http import HTTPSession
+
+from .models import Event, EventResponse, Resource, Metadata
+
+API = "https://app.pendo.io/api/v1"
+
+
+async def fetch_resources(
+        http: HTTPSession,
+        entity: str,
+        log: Logger,
+) -> Iterable[Resource]:
+    url = f"{API}/{entity}"
+
+    response = json.loads(await http.request(log, url))
```

Comment:
The `json.loads` and `json.dumps` indicate a mismatch in how the API is presenting data vs. how we are modeling it. Fortunately there's a pretty easy way to resolve this I think...if we're expecting a list of `Resource`, I think you can use a pydantic `TypeAdapter` to clean this up, something like:

```
TypeAdapter(list[Resource]).validate_json(await http.request(...))
```

Reply:
`TypeAdapter` works, thank you!